build.gradle

```diff
@@ -1,17 +1,23 @@
 ext {
     GROUPID = 'de.golfgl.gdxgamesvcs'
     VERSION = '1.1.0'
-
 }
 
 buildscript {
     repositories {
-        maven { url 'https://plugins.gradle.org/m2/' }
+        mavenLocal()
+        mavenCentral()
         google()
+        maven { url "https://plugins.gradle.org/m2/" }
+        maven { url "https://oss.sonatype.org/content/repositories/snapshots/" }
+        maven { url 'https://developer.huawei.com/repo/' }
+        gradlePluginPortal()
```

Comment:
I think this is the same as plugins.gradle.org/m2

Reply:
Yeah I didn't realize this. I will remove line 11 as gradlePluginPortal is already defined.